main.py

```diff
@@ -23,15 +23,13 @@
 
 PLUGIN_NAME = 'LSP'
 SUBLIME_WORD_MASK = 515
-MARKUP_ERROR = 'markup.error.lsp sublimelinter.mark.error'
 show_status_messages = True
 show_view_status = True
 auto_show_diagnostics_panel = True
 show_diagnostics_phantoms = True
```

Comment:
For consistency's sake, can you change this value to the settings default of True?